src/cmdlib.sh

```diff
@@ -883,6 +907,9 @@ prepare_git_artifacts() {
 }
 EOC
     chmod 0444 "${json}"
+    if [ "$delete_tarball" != "false" ]; then
```

Comment:
Instead of this, let's instead skip tarring entirely in the first place and also not add the `file` key in the JSON.

Reply:
Ye that sounds good :+1: